tests/e2e/specs/factory/Factory.spec.ts

```diff
@@ -45,7 +46,7 @@ suite(
 		const projectAndFileTests: ProjectAndFileTests = e2eContainer.get(CLASSES.ProjectAndFileTests);
 		const driverHelper: DriverHelper = e2eContainer.get(CLASSES.DriverHelper);
 		const loginTests: LoginTests = e2eContainer.get(CLASSES.LoginTests);
-		const webCheCodeLocators: Locators = e2eContainer.get(CLASSES.CheCodeLocatorLoader);
+		const webCheCodeLocators: Locators = new CheCodeLocatorLoader().webCheCodeLocators;
```

Comment:
Because differently, it doesn't work the `openItem`..
Take a look on the `factory/NoSetupRepoFactory.spec.ts` -> https://github.com/eclipse/che/blob/main/tests/e2e/specs/factory/NoSetupRepoFactory.spec.ts#L49

and 
`factory/RefusedOAuthFactory.spec.ts` -> https://github.com/eclipse/che/blob/main/tests/e2e/specs/factory/RefusedOAuthFactory.spec.ts#L48

There works well
I copied it.

Reply:
Could you fix it everywhere, please? =)

Reply:
Fixing is done. 
Thanks for the tip.